src/mono/wasm/runtime/exports.ts

```diff
@@ -106,26 +107,50 @@ export const BINDING: BINDING = <any>{
     _teardown_after_call,
 };
 
+export declare function createDotnetRuntime(Module: EmscriptenModuleConfig): Promise<DotNetExports>;
+
 // this is executed early during load of emscripten runtime
 // it exports methods to global objects MONO, BINDING and Module in backward compatible way
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-function export_to_emscripten(dotnet: any, mono: any, binding: any, internal: any, module: any): void {
+export const __exportAPI: any = (mono: any, binding: any, internal: any, module: any): void => {
```

Comment:
this syntax is really weird, can we keep it as a function and then expose the function as __exportAPI separately